### DISCUSSION TITLE
Can’t able to join slack community

### DISCUSSION THREAD

**Discussion by Rahul-Kumar-prog** [gruntwork-io/knowledge-base#851] [Help]
Hello gruntwork team, 
I want to join the slack channel but I couldn't able to join the slack? 
I don't know why.

Can you please sent me the invitations to joint slack please. On the given email 

rahulkumarrsde@gmail.com

**Comment by kkrav3ts** [comment]
Please use [join link](https://gruntwork-community.slack.com/archives/CHH9Y3Z62). Also, there are other ways to [get support](https://docs.gruntwork.io/support/) if needed.
